config/routes.rb

```diff
@@ -2,8 +2,10 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   root to: redirect("/step-by-step-pages", status: 302)
-  post "/coronavirus/publish_live_stream", to: "coronavirus#publish_live_stream"
   get "/coronavirus/live_stream", to: "coronavirus#live_stream"
+  post "/coronavirus/update_live_stream", to: "coronavirus#update_live_stream"
```

Comment:
Convention might be for this to be a `put`? (See the `rake routes` output for the coronavirus resources below)

Reply:
thanks